package.json

```diff
@@ -207,7 +207,7 @@
   },
   "scripts": {
     "----------------------------------------- static analysis -----------------------------------------": "",
-    "lint": "eslint lib test --quiet --fix",
+    "lint": "eslint lib test *.[tj]s --quiet --fix",
```

Comment:
This would still limit eslint to the lib and test folder right? Don't think this actually changes anything

Reply:
Yes, it limits to the test and lib folders plus all ts+js files in the root, like index.js and build.js.

Reply:
While we're changing this, can you include `m` & `c` prefix for mjs, cjs, mts, & cts?